lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -391,33 +403,30 @@
         </div>
       </div>
 
-      <?php // Voter Registration Modal // ?>
-      <?php if (array_key_exists('enable_voter_registration', $campaign->variables) && $campaign->variables['enable_voter_registration']) : ?>
-      <?php // @TODO: if user is not registered and is eligible, pop this up ?>
-        <?php if ($can_vote) : ?>
-          <div data-modal id="modal-voter-registration" role="dialog">
-            <div class="floatbox" data-fb-options="width:618 height:max scrolling:yes">
-              <iframe src="https://register2.rockthevote.com/registrants/map/?source=iframe&partner=35164" width="100%" height="1200" marginheight="0" frameborder="0"></iframe>
-            </div>
-          </div>
-        <?php else : ?>
-          <?php // @TODO: fix the styling in hurr ?>
+
+        <?php // Voter Registration Modal // ?>
+        <?php if (dosomething_campaign_feature_on($campaign, 'enable_voter_registration')) : ?>
           <div data-modal id="modal-voter-registration" role="dialog">
+            <iframe src="https://register2.rockthevote.com/registrants/map/?source=iframe&partner=35164" width="100%" height="1000" marginheight="0" frameborder="0"></iframe>
```

Comment:
might be worth sending the modal/signup form to the theme so we don't have to repeat this chuck here and above